server/middleware.go

```diff
@@ -172,6 +179,21 @@ func (server *Server[state]) SessionMiddleware() func(http.Handler) http.Handler
 				ctx = context.WithValue(ctx, ctxUserID, userID)
 			}
 
+			organisationID, ok := session.Values[sessionOrganisationID].(core.OrganisationID)
```

Comment:
deze code is wel heel veel herhaling. Heeft golang hiervoor geen generics die toelaten om dit in een loop te schrijven? Iets als:
```
function updateContext<T>(session, ctx, varName, ctxVarName) {
    val, ok := session.Values[varName].(T)
    if ok {
        return context.WithValue(ctx, ctxVarName, val)
    }
    return ctx
}

Reply:
Je zou dat kunnen maar ik denk dat dat meer bugs zou introduceren dan het expliciet in-line te doen. Het is ook niet altijd hetzelfde, sommige zaken zijn pointers (bv parentID) en die kan je uiteraard niet als pointer opslaan, dus die moet je omzetten tussen beide